src/ApiPort.VisualStudio/AnalysisOptions.cs

```diff
@@ -24,12 +21,12 @@ public class AnalysisOptions : IApiPortOptions
         public string OutputFileName { get; private set; }
         public IEnumerable<string> IgnoredAssemblyFiles { get { return Enumerable.Empty<string>(); } }
 
-        public AnalysisOptions(string description, IEnumerable<string> inputAssemblies, IEnumerable<string> targets, string format, bool discardMetadata, string outputFileName)
+        public AnalysisOptions(string description, IEnumerable<string> inputAssemblies, IEnumerable<string> targets, IEnumerable<string> formats, bool discardMetadata, string outputFileName)
         {
             Description = description;
             InputAssemblies = inputAssemblies.Select(target => new AssemblyFile(target)).ToList();
             Targets = targets.ToList();
-            OutputFormats = new[] { format };
+            OutputFormats = formats.ToList();
```

Comment:
I'm gonna keep this one and remove the other. At this point, the formats (and the other enumerables) should be realized so that they don't change at any point.